.woodpecker.yml

```diff
@@ -61,10 +61,8 @@ steps:
       - event: pull_request
 
   sql_fmt:
-    image: debian:bookworm
+    image: backplane/pgformatter:latest
     commands:
-      - apt-get update
-      - apt-get --yes --no-install-recommends --no-install-suggests install pgformatter
```

Comment:
Dont change this, otherwise our ci will break without warning again when there is a new version of pgformatter.

https://github.com/LemmyNet/lemmy/pull/5521

Reply:
I think I'd rather use the backplane image. The version in the debian repos is > 5 years old. My arch install is also correctly update to the same version thats in the backplane image, `5.6`

The releases are rare enough that I think its better for CI to catch it anyway.

Reply:
That's 5 years old though, and many versions behind, we shouldn't be using it. 

I'll try to do this with alpine and perl, instead of using the backplane image tho.

Reply:
K got it versioned now.